pages/pipelines/command_step.md

```diff
@@ -214,7 +214,10 @@ To prevent jobs from consuming too many job minutes or running forever, you can
 
 Specific timeouts take precedence over more general ones — a step level timeout takes precedence over a pipeline timeout, which in turn takes precedence over an organization default.
 
-Maximum timeouts are applied to any command step without a timeout or with a timeout greater than the maximum timeout.
+Maximum timeouts are applied to command steps in the following situations:
```

Comment:
Style: You need to include a blank line before a list for it to render correctly. At the moment, it looks like:

<img width="741" alt="Screenshot 2023-04-20 at 8 20 54 am" src="https://user-images.githubusercontent.com/119824349/233212370-4ab8f0f5-a778-4278-9685-4ba3580592ca.png">

```suggestion
Maximum timeouts are applied to command steps in the following situations:

```